src/main/kotlin/org/koil/admin/AdminService.kt

```diff
@@ -16,9 +20,10 @@ import org.springframework.stereotype.Component
 import org.springframework.transaction.annotation.Transactional
 
 interface AdminService {
-    fun createAdminFromEmail(email: String, password: HashedPassword): UserCreationResult
 
-    fun getAccounts(queryingAsAccount: Long, pageable: Pageable): Page<Account>
+    fun createDefaultAdminOrganization(organizatioName: String): OrganizationCreatedResult
```

Comment:
I removed it as a method in the AdminService Interface at one point - and it also should be private. I don't know how it got back in - I've removed it again and made it private. I thought since it doesn't get called from anywhere else in the project that we shouldn't bother creating an interface for it.